packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -91,10 +109,15 @@ describe('<AddressDetails/>', () => {
     });
 
     it('should render AddressDetails component for mobile', async () => {
-        (isDesktop as jest.Mock).mockReturnValue(false);
-        (isMobile as jest.Mock).mockReturnValue(true);
+        const new_store_config = {
+            ...store,
+            ui: {
+                ...store.ui,
+                is_mobile: true,
+            },
+        };
```

Comment:
```suggestion
        store.ui.is_mobile = true,
```
Could you check it? I think we can just change the value without creating new store

Reply:
We can do it. I feel using immutable data is better